Modifications in the Footer

Abhishek-465 opened the pull request:
# Modifications in the Footer
# Fixes #32 

I have modified the footer as per the reference provided. I have also made it responsive.

# Video:

https://github.com/akash0708/ieee-jusb-new-web/assets/127030695/7fb5426a-f8ac-4624-923b-0e36abea425f

Abhishek-465 committed:
Update Footer.jsx

Abhishek-465 committed:
Create Footer.css

akash0708 commented:
remove the hover effect and add underlines as shown in the original design in issue #32

Abhishek-465 committed:
Merge branch 'master' into Footer_update

Abhishek-465 commented:
@akash0708  Okay doing it

Abhishek-465 committed:
Update Footer.css

Abhishek-465 commented:
@akash0708 
Removed the hover effect and added underlines by default.

![Screenshot (11)](https://github.com/akash0708/ieee-jusb-new-web/assets/127030695/ddcc2b86-4a50-4b6f-b3a1-e65faaca6975)
